spec/Clay/SelectorSpec.hs

```diff
@@ -0,0 +1,15 @@
+{-# LANGUAGE OverloadedLists   #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Clay.SelectorSpec where
+
+import Clay
+import Common
+import Test.Hspec
+
+spec :: Spec
+spec = describe "Selector with filtering" $
+    describe "[x # (y ^= z)]" $
+      ".row[class^=\"col\"]" `shouldRenderAsFrom`
```

Comment:
Either simplify with `shouldRenderFrom` or add the space after `row` (which is the whole point behind the PR, right?)

Reply:
I will update the PR if we decide to go further with it, for now, I don't see a point to work further on it until you can confirm that the fix (in any form) is desirable in here :slightly_smiling_face: